### PR TITLE
further updates to demo calculation

### PR DESCRIPTION
**Pull request by ltalirz**
* shorten instructions
 * put some actual questions for people to answer

### CODE REVIEW

**Comment by ltalirz** [comment]
@giovannipizzi just fyi here are my updates

File: docs/pages/2019_Xiamen/sections/first_taste.rst [hunk 1]
```diff
@@ -70,49 +71,37 @@ Again, your calculation will get a PK, which you can use to get more information
...
-You might wonder where is the actual input file of Quantum ESPRESSO. This has been generated automatically by AiiDA and
-one way to look to it is:
-
-.. code:: bash
-
-   verdi calcjob inputcat <PK>
-
-As an exercise, compare the Quantum ESPRESSO input file with the python file that you run, to understand how the input
-provided in it (in "AiiDA" format) has been converted into the Quantum ESPRESSO input.
+Try to reproduce the figure using the PK of your calculation.
```
Comment (giovannipizzi) [inline docs/pages/2019_Xiamen/sections/first_taste.rst:88]: I would have left a note to wait for the calculation to finish somewhere.

**Review by giovannipizzi — APPROVED** [review]
apart for my comment, good to go

File: docs/pages/2019_Xiamen/sections/first_taste.rst (hunk 1 above)
Comment (ltalirz) [reply docs/pages/2019_Xiamen/sections/first_taste.rst:88]: I'd say it's actually good if they produce the figure and it doesn't yet contain the outputs, this way they can think a little bit.
I'm pretty sure they can figure that one out by themselves ;-)

**Pull request merged**